tests/system/Router/RouterTest.php

```diff
@@ -197,35 +197,47 @@ public function testClosures(): void
     public function testAutoRouteFindsDefaultControllerAndMethod(): void
     {
         $this->collection->setAutoRoute(true);
-        $this->collection->setDefaultController('Test');
-        $this->collection->setDefaultMethod('test');
+        $this->collection->setDefaultController('Mycontroller');
+        $this->collection->setDefaultMethod('getSomemethod');
         $router = new Router($this->collection, $this->request);
 
+        copy(TESTPATH . 'system/Router/Controllers/Mycontroller.php', APPPATH . 'Controllers/Mycontroller.php');
```

Comment:
Sorry, this way is not good. Because the namespace is wrong if it is copied to `App\Controllers`.

Reply:
Do you have any idea to fix this error? 😢

Reply:
Create controller files somewhere else and set the namespaces to `App\Controllers`, and copy them to `APPPATH . 'Controllers/`.

Reply:
I create the `App` folder under the mock controller in the latest commit.